docs/guides/integration-javascript-client-side.md

```diff
@@ -260,24 +262,28 @@ window.__euid.callbacks.push(async (eventType, payload) => {
         var advertising_token_to_use = payload.identity.advertising_token;
       } else {
           if (__euid.isLoginRequired()) {
-            // Call one of the setIdentityFrom functions to generate a new EUID token.
-            // Add any retry logic around this call as required.
-            await __euid.setIdentityFromEmailHash(
-                emailHash,
-                clientSideConfig
-          );
+              // Call one of the setIdentityFrom functions to generate a new EUID token.
+              // Add any retry logic around this call as required.
+              await __euid.setIdentityFromEmailHash(
+              emailHash,
+              clientSideConfig);
+          }  
           else {
-            // there is a token generation API call in flight which triggers
-            // a IdentityUpdated event 
+              // there is a token generation API call in flight which triggers a IdentityUpdated event 
+              // or no token would be generated because one of previous `setIdentity` calls determines the user's personal data has opted out.
           }
-        }
       }
       break;
- 
+      
     case "IdentityUpdated":
       // The IdentityUpdated event happens when an EUID token is generated or refreshed.
       // See previous comment for an example of how the payload looks.
-      var advertising_token_to_use = payload.identity.advertising_token;
+      // It's possible that payload/identity objects could be null for reasons such as the token
+      // expired and cannot be refreshed, or the user opted out of EUID. 
```

Comment:
// expired and cannot be refreshed, or the user opted out of EUID.
Update to
// has expired and cannot be refreshed, or the user has opted out of EUID.

Reply:
Made this edit in branch for update per Sunny's request.